srcpkgs/GammaRay/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'GammaRay'
+pkgname=GammaRay
+version=2.11.1
+revision=1
+wrksrc="gammaray-${version}"
+build_style=cmake
+configure_args="-DGAMMARAY_INSTALL_QT_LAYOUT=true -DECM_MKSPECS_INSTALL_DIR=/usr/lib/qt5/mkspecs/modules -DPLUGIN_INSTALL_DIR=/usr/lib/qt5/plugins/gammaray"
+hostmakedepends="qt5-host-tools qt5-devel"
+makedepends="qt5-devel"
+short_desc="Tool to poke around in a Qt-application"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/KDAB/GammaRay"
+distfiles="${homepage}/releases/download/v${version}/gammaray-${version}.tar.gz"
+checksum=87a1d72ad1ad6d1a0156c54a85b0976ab38c6a64136458ca7c4ee491566d25d0
+
+LDFLAGS+=" -Wl,-no-fatal-warnings"
+
+pre_build() {
```

Comment:
Should be `pre_configure` or I'd expect the `vsed` substitution below to accumulate in incremental builds, i.e. if a `xbps-src pkg` terminates during the `build` phase, on the next run the result will probably be `lib/qt5/lib/qt5/plugins/gammaray`, etc.

In clean builds, this would save the second `cmake` run when it detects that the files have changed.

Reply:
Good suggestion.
But `configure` sometimes fails, too. Let's say in next version, they add new dependencies.

For that reason, I prefer to use `post_patch` instead.
It matches with the action: we're patching the code.

Reply:
Ah, of course, that's even better!